image/util.go

```diff
@@ -110,3 +111,22 @@ func GetClusterFileFromBaseImage(imageName string) string {
 	}
 	return string(data)
 }
+
+func getDockerAuthInfoFromDocker(domain string) types.AuthConfig {
```

Comment:
I do not think that making username/password empty when failure happens is reasonable. It is not so friendly for users.